### ISSUE TITLE
ANI (full?) percentage identity

### ISSUE THREAD

**Issue by cdiazmun** [merenlab/anvio#1505]
Good morning everyone!

I had a dataset with >100 strains from the same species and I performed both a Phylogenomic and a Pangenome analysis. In the pangenome, I clustered my genomes according to `gene_cluster frequencies` and it gave me the best results (it made sense to me the way they were orginizing and it was in lane with both the phylogenomic tree and literature). I included then the ANI results and when I add the layer `ANI_percentage_identity` I was very happy, as it was again making sense, but when I included the `ANI_full_percentage_identity`... it differs a lot from the clustering. `ANI_full_percentage_identity` was actually more similar to the clustering made by `gene_cluster presence absence` way of ordering the layers. 

So basically my question is: 

What is the difference between the calculated identities between the two files? 

I tried to watch it out at Pritchard's paper and pyANI GitHub page, but I didn't found any answer to that. Sorry to bother and thank you in advance for any help!

**Comment by cdiazmun** [comment]
Interesting: with `ANI_full_percentage_identity` the strains are clustering according to the "authorship" (if I took 20 strains sequenced by _Example et al., 2020_ they're all clustered together). So the sequencing run and/or assembly strategy played an important role...but I only see this with the `ANI_full_percentage_identity` and not `ANI_percentage_identity` or the phylogenomic tree (based on single copy genes). Does `ANI_percentage_identity` only take a subset of the genome to infer the ANI and  `ANI_full_percentage_identity` the entire genome?

**Comment by ekiefl** [comment]
@cdiazmun I think you should look at the alignment fractions. This parameter in the help menu of `anvi-compute-genome-similarity`:

```
  --min-full-percent-identity FULL_PERCENT_IDENTITY
                        In some cases you may get high raw ANI estimates
                        (percent identity scores) between two genomes that
                        have little to do with each other simply because only
                        a small fraction of their content may be aligned. This
                        can be partly alleviated by considering the *full*
                        percent identity, which includes in its calculation
                        regions that did not align. For example, if the
                        alignment is a whopping 97 percent identity but only 8
                        percent of the genome aligned, the *full* percent
                        identity is 0.970 * 0.080 = 0.078 OR 7.8 percent.
                        *full* percent identity is always included in the
                        report, but you can also use it as a filter for other
                        metrics, such as percent identity. This filter will
                        set all ANI measures between two genomes to 0 if the
                        *full* percent identity is less than you deem
                        trustable. When you set a value, anvi'o will go
                        through the ANI results, and set all ANI measures
                        between two genomes to 0 if the *full* percent
                        identity *between either of them* is less than the
                        parameter described here. The default is 0.
```

I hope this make sense. Since this is not an issue, but a feature, I am closing this issue, but feel free to comment if you're still confused.
